frontend/mainwindow.h

```diff
@@ -67,6 +67,9 @@ class MainWindow : public QMainWindow
     QMenu *runsMenu_;
     QMenu *contextMenu_;
     JsonTableModel::Header header_;
+    QList<QString> desiredHeader_;
+    QList<QString> muonHeader_;
+    QList<QString> neutronHeader_;
```

Comment:
As we discussed, the default displayed headers could be stored in our main XML file on a per-instrument basis, opening up the door for some nice configuration options.